spec/system/provider_interface/provider_views_application_submitted_in_new_cycle_feature_flag_disabled_spec.rb

```diff
@@ -1,185 +0,0 @@
-require 'rails_helper'
-
-RSpec.feature 'Provider views application submitted in new cycle' do
-  include CandidateHelper
-  include CourseOptionHelpers
-  include DfESignInHelpers
-
-  before do
-    TestSuiteTimeMachine.travel_permanently_to(CycleTimetable.apply_opens(ApplicationForm::OLD_REFERENCE_FLOW_CYCLE_YEAR - 1) + 1.day)
-  end
-
-  it 'but started in the previous one' do
-    given_the_feature_flag_is_disabled
-    given_i_am_signed_in_as_a_candidate
-    when_i_have_an_unsubmitted_application_without_a_course
-    and_the_new_recruitment_cycle_begins
-
-    when_i_sign_in_again
-    and_i_visit_the_application_dashboard
-    then_i_cannot_submit_my_application
-    and_i_am_redirected_to_the_carry_over_interstitial
-
-    when_i_click_on_continue
-    then_i_see_a_copy_of_my_application
-
-    when_i_view_referees
-    then_i_can_see_the_referees_i_previously_added
-
-    when_i_view_courses
-    then_i_can_see_that_i_need_to_select_courses
-
-    and_i_select_a_course
-    and_i_complete_the_section
-    and_i_receive_references
-    and_i_submit_my_application
-    and_my_application_is_awaiting_provider_decision
-
-    given_i_am_a_provider_user_with_dfe_sign_in
-    and_i_am_permitted_to_see_the_candidates_application
-    and_i_sign_in_to_the_provider_interface
-
-    when_i_visit_the_provider_page
-    then_i_can_see_a_filter_for_the_current_recruitment_cycle_year
-    then_i_can_see_and_load_the_candidates_application
-  end
-
-  def given_the_feature_flag_is_disabled
-    FeatureFlag.deactivate(:one_personal_statement)
```

Comment:
Created a card to look at these old feature flags: https://trello.com/c/eus6FksO/1084-tech-debt-clean-up-feature-flags